cfgov/jinja2/v1/know-before-you-owe/compare/index.html

```diff
@@ -106,11 +105,11 @@ <h3>Initial TIL disclosure + Good Faith Estimate</h3>
             <h2>After</h2>
             <h3>Loan Estimate</h3>
             <div class="kbyo-compare-tiles kbyo-compare-tiles-overlay">
-                <a href="https://files.consumerfinance.gov/f/201311_cfpb_le_page1.png" data-lightbox="new-loan-estimate" title="Loan Estimate page 1">
+                <a href="https://files.consumerfinance.gov/f/201311_cfpb_le_page1.png" title="Loan Estimate page 1">
                 <img src="https://files.consumerfinance.gov/f/201311_cfpb_le_small_page1.png" alt="Loan Estimate page 1" title="Click to enlarge"> </a>
-                <a href="https://files.consumerfinance.gov/f/201311_cfpb_le_page2.png" data-lightbox="new-loan-estimate" title="Loan Estimate page 2">
+                <a href="https://files.consumerfinance.gov/f/201311_cfpb_le_page2.png" title="Loan Estimate page 2">
                 <img src="https://files.consumerfinance.gov/f/201311_cfpb_le_small_page2.png" alt="Loan Estimate page 2" title="Click to enlarge"> </a>
-                <a href="https://files.consumerfinance.gov/f/201311_cfpb_le_page1.png" data-lightbox="new-loan-estimate" title="Loan Estimate page 3">
+                <a href="https://files.consumerfinance.gov/f/201311_cfpb_le_page1.png" title="Loan Estimate page 3">
```

Comment:
```suggestion
                <a href="https://files.consumerfinance.gov/f/201311_cfpb_le_page3.png" title="Loan Estimate page 3">
```

It looks like the image for page 3 is currently linking to page 1 -- maybe we can add a fix for that here?